tracer/src/Datadog.Trace/Configuration/ConfigurationSources/JsonConfigurationSource.cs

```diff
@@ -44,9 +53,12 @@ public JsonConfigurationSource(string json)
         /// <param name="filename">A JSON file that contains configuration values.</param>
         /// <returns>The newly created configuration source.</returns>
         public static JsonConfigurationSource FromFile(string filename)
+            => FromFile(filename, ConfigurationOrigins.Code);
```

Comment:
`ConfigurationOrigins.Code` for a json file? (also above in `ctor(string)`)

Reply:
So this is a bit of a tricky one... These are the public constructors, so these should _only_ be called by customers... and in that case it's _kind_ of configuration in Code, right? 

The only place _we_ use the `JsonConfigurationSource` currently, we mark the origin as `DdConfig` as you'd expect